sdk/core/azure-core-tracing-opentelemetry/setup.py

```diff
@@ -57,9 +56,9 @@
     packages=[
         'azure.core.tracing.ext.opentelemetry_span',
     ],
-    python_requires=">=3.5.0",
+    python_requires=">=3.6.0",
     install_requires=[
-        'opentelemetry-api==0.17b0',
+        'opentelemetry-api==1.0.0',
```

Comment:
Why are we pinning an exact version?

Reply:
so far we have pinned since it was in beta and things were breaking randomly - let's keep it this way till monitor exporter is released and we come up with a plan on how to keep both versions identical

Reply:
We pin preview versions since they don't follow semver. We don't pin GA versions for libraries that do follow semver since that locks the application to a very specific version. Do we have specific reasons to believe that OpenTelemetry will not follow semver?

Reply:
no - OT will follow the semver - un-pinned it